ui/src/views/settings/SettingsView.tsx

```diff
@@ -225,6 +225,22 @@ export default class SettingsView extends React.Component<Props> {
                 onChange={value => this.updateSetting('openWindowOnStartup', value)}
               />
             </Form.Item>
+            <Form.Item label="Force quit on close">
```

Comment:
nit: 'Quit PostyBirb on close' or some such might be better terminology than 'Force quit'

Reply:
Done!